### DISCUSSION TITLE
[Suggestion] Level selecting/entering screens need help text

### DISCUSSION THREAD

**Discussion by adem4ik** [SuperTux/supertux#2104] [Ideas]
<sub>Note: Edited by Semphris to remove issue formatting</sub>

It was very unintuitive for me when I've entered the world for the first time. I guess some people may stuck at those screens too. So it would be very helpful if the screens had some help messages with explanation like:
- "Use arrows to select the level. Press Enter to start the level"
- "Press Enter to start the game" 

##### Suggestions
- Start game - Story mode - Select the level -> there should be some kind of the text
- Select the level - Enter the black screen -> there should be some kind of the text

![image](https://user-images.githubusercontent.com/4707112/149668417-cb21f753-f775-4cb9-ac7f-cbfa6d8f96d4.png)
![image](https://user-images.githubusercontent.com/4707112/149668445-626840c8-c188-4198-bf4c-0db8fb1a8016.png)

**Comment by Semphriss** [comment]
SuperTux made it 20+ years in development without this being an issue, it's really not everybody who has this issue. **However**, considering the Steam and Google Play releases, it is to be expected that people with the least amount of experience with gaming in general will discover SuperTux (which, IMO, is a good thing, but will indeed require adaptation). I'm not sure if SuperTux should give instructions all the time like NSMBW, but perhaps a tutorial of some sort when the game is first launched, a bit à la STK, could be interesting.
